doc/source/release-notes/Percona-Server-5.6.36-82.1.rst

```diff
@@ -20,43 +20,83 @@ milestone at Launchpad
 New Features
 ============
 
- 
+|Percona Server| can now be built with support of OpenSSL 1.1.
+
+|Percona Server| is now available on Debian 9 (stretch). The support only covers
+the ``amd64`` architecture.
+
+|TokuDB| enables to kill a query that is awaiting an FT locktree lock.
+
 
 Bugs Fixed
 ==========
 
- |Percona Server| could crash when running a data lookup on the partition table
- if there was only one partition over the time period following simple test. Bug
- fixed :bug:`1657941` (upstream :mysqlbug:`76418`).
+ Row counts in |TokuDB| could be lost intermittently after restarts. Bug fixed
+ :tdbbug:`2`.
+
+ In |TokuDB|, two races in the fractal tree lock manager could significantly
+ affect transactional throughput for some applications that used a small number
+ of concurrent transactions.  These races manifested as transactions
+ unnecessarily waiting for an available lock. Bug fixed :tdbbug:`3`.
+
+ |Percona FT| could assert when opening a dictionary with no useful information
+ to error log. Bug fixed :tdbbug:`23`.
+
+ |Percona FT| could assert for various reasons deserializing nodes with no useful
+ error output. Bug fixed :tdbbug:`24`.
+
+ |Percona Server| could crash when running a query over a partitioned table that
+ uses an index to read a range of rows if this range was not covered by any
+ existing partition. Bug fixed :bug:`1657941` (upstream :mysqlbug:`76418`).
 
  With two client connections to a server (debug server build), the server could
  crash after one of the clients set the global option ``userstat`` and flushed
- the client statistics (``FLUSH CLIENT_STATICTICS``) and then both clients were
+ the client statistics (``FLUSH CLIENT_STATISTICS``) and then both clients were
  closed. Bug fixed :bug:`1661488`.
 
- When connecting to the Aurora cluster end point using SSL, SAN (Subject
- Alternative Name) certificates were ignored.  Bug fixed :bug:`1673656` (upstream
- :mysqlbug:`68052`).
+ |Percona FT| did not pass cmake flags on to snappy cmake. Bug fixed
+ :tdbbug:`41`.  The progress status for partitioned TokuDB table ALTERs was
+ misleading. Bug fixed :tdbbug:`42`.
 
- Killing a stored procedure which contained a query with a view resulted in a
- debug assertion failure. Bug fixed :bug:`1689736` (upstream :mysqlbug:`86260`).
+ When a client application connecting to the Aurora cluster end point using SSL
```

Comment:
Previous suggestion not implemented? (no need to mention Aurora; Yura's OpenSSL version comment not added)